payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -90,16 +89,15 @@ internal abstract class BaseAddPaymentMethodFragment(
 
         childFragmentManager.addFragmentOnAttachListener { _, fragment ->
             (fragment as? ComposeFormDataCollectionFragment)?.let { formFragment ->
-                formFragment.formViewModel.completeFormValues.map { formFieldValues ->
-                    transformToPaymentSelection(
-                        formFieldValues,
-                        formFragment.formSpec.paramKey,
-                        selectedPaymentMethod
-                    )
-                }
-                    .asLiveData()
-                    .observe(viewLifecycleOwner) { paymentSelection ->
-                        sheetViewModel.updateSelection(paymentSelection)
+                formFragment.formViewModel.completeFormValues.asLiveData()
```

Comment:
Is this a style change or a result of a bug?

Reply:
Style change